src/palace/manager/api/sirsidynix_authentication_provider.py

```diff
@@ -194,22 +193,30 @@ def remote_authenticate(
 
     def remote_patron_lookup(
         self, patron_or_patrondata: Patron | PatronData
-    ) -> None | SirsiDynixPatronData:
+    ) -> None | SirsiDynixPatronData | PatronData:
```

Comment:
Since `SirsiDynixPatronData` is a `PatronData` this can be simplified to `None | PatronData`